src/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -56,3 +57,15 @@ uiRoutes
       }
     }
   });
+
+FeatureCatalogueRegistryProvider.register(() => {
+  return {
+    id: 'dashboard',
+    title: 'Dashboards',
+    description: 'Create visual landing pages made up of content from other apps',
+    icon: '/plugins/kibana/assets/app_dashboard.svg',
+    path: '/app/kibana#/dashboard',
```

Comment:
What about creating this url in `dashboard_constants`? we already have a `createDashboardEditUrl` and then you can take advantage of the constant `LANDING_PAGE_PATH` which I think is what we want here instead of the singular dashboard, the plural.  If you use the singular it may sometimes direct to the last opened dashboard.  That's not terrible, but it's inconsistent with visualize because visualize specifically links to the landing page path.

The reason this is so confusing, the singular vs the plural, is due to BWC issues.  heres a comment left on the kibana index registering dashboard as an app:
```
        }, {
          id: 'kibana:dashboard',
          title: 'Dashboard',
          order: -1001,
          url: `${kbnBaseUrl}#/dashboards`,
          // The subUrlBase is the common substring of all urls for this app. If not given, it defaults to the url
          // above. This app has to use a different subUrlBase, in addition to the url above, because "#/dashboard"
          // routes to a page that creates a new dashboard. When we introduced a landing page, we needed to change
          // the url above in order to preserve the original url for BWC. The subUrlBase helps the Chrome api nav
          // to determine what url to use for the app link.
          subUrlBase: `${kbnBaseUrl}#/dashboard`,
          description: 'compose visualizations for much win',
          icon: 'plugins/kibana/assets/dashboard.svg',
        }
```

So what we are using for the other links is the url, which is probably what we should be using here too.